OrbitGl/App.cpp

```diff
@@ -345,12 +345,14 @@ void OrbitApp::LoadFileMapping() {
 
       bool containsQuotes = absl::StrContains(line, "\"");
 
-      std::vector<std::string> tokens = Tokenize(line);
+      std::vector<std::string> tokens = absl::StrSplit(line, ' ');
+
       if (tokens.size() == 2 && !containsQuotes) {
         m_FileMapping[ToLower(tokens[0])] = ToLower(tokens[1]);
       } else {
         std::vector<std::string> validTokens;
-        for (const std::string& token : Tokenize(line, "\"//")) {
```

Comment:
The "Tokeninze" function could take multiple characters as arguments, like in this particular case.  We should probably use absl::ByAnyChar("\"//") to keep the same behavior here.

Reply:
I initially did that but "//" seems very suspicious, these are 2 characters. I think this is supposed to filter out comments but in fact this is '/' or '/' which does not make much sense - so I removed this.

Another thing is this code only takes into consideration 2 first tokens, which is why it works I suppose. Also for some reason applies only on strings with '"'  was this intentional?

I can just implement stripping comment at the end of line separately.

Reply:
Oh ok, it is indeed very suspicious.  I'd say leave it as is then, I don't even think the idea was to strip end of line comment.